roles/custom/matrix-prometheus-nginxlog-exporter/defaults/main.yml

```diff
@@ -0,0 +1,46 @@
+---
+# prometheus-nginxlog-exporter exports nginx logs into a prometheus usable format on a `/metrics/ endpoint
+# See: https://github.com/martin-helmich/prometheus-nginxlog-exporter/
+
+matrix_prometheus_nginxlog_exporter_enabled: true
+matrix_prometheus_nginxlog_exporter_container_url: 'matrix-prometheus-nginxlog-exporter'
```

Comment:
Maybe `matrix_prometheus_nginxlog_exporter_container_hostname` is a better variable name?

Reply:
agreed, done